LuaCppInterface/luacppinclude.h

```diff
@@ -2,7 +2,7 @@
 #define LUACPPINCLUDE
 
 
-extern "C" {
+extern "C++" {
 #include "lua.h"
```

Comment:
this causes the headers to be compiled as C++ code, mangling their names. the Lua C lib built in C will have different names. Perhaps you need to compile your Lua code on windows in as C code.